tests/e2e/requirements.txt

```diff
@@ -1,12 +1,12 @@
-boto3==1.18.50
-kfp==1.8.9
-kubernetes==12.0.1
-pytest==6.2.5
-PyYAML==5.4
-black==22.8.0
-mysql-connector-python==8.0.27
-retrying==1.3.3
-requests==2.28.1
-numpy==1.23.3
-sagemaker==2.0.0
-pandas==1.5.0
\ No newline at end of file
+boto3==1.26.117
```

Comment:
since this is being used by various scripts. Can we run one cognito-rds-s3 installation before merging?

Reply:
how about I revert it back to 1.18.50 first so we can merge this and I will test the new version offline.

Reply:
I meant the whole file, not just boto3

Reply:
tested. all working